docs/components/pagination/index.html

```diff
@@ -0,0 +1,146 @@
+---
+title: Pagination
+also:
+  components/buttons: Buttons
+---
+{% extends 'component.njk' %}
+{% block content %}
+  <section>
+    <h2 class="hxSectionTitle" id="first-page">First Page</h2>
+    <div class="demo">
+      <div class="hxPagination hxBtnGroup">
+        <button class="hxBtn" disabled>
+          <hx-icon type="angle-start"></hx-icon>
+        </button>
+        <button class="hxBtn" disabled>
+          <hx-icon type="angle-left"></hx-icon>
+        </button>
+        <button class="hxBtn" aria-current="true">1</button>
```

Comment:
We need a blurb somewhere to note that the `aria-current` attribute needs added to apply style as well as denote semantic meaning. Probably at the top of the page, below the page title (similar to Text Inputs).